entries/p/passpack.com.json

```diff
@@ -1,14 +1,17 @@
 {
   "Passpack": {
     "domain": "passpack.com",
-    "url": "https://www.passpack.com/",
     "tfa": [
-      "email",
+      "totp",
       "custom-hardware"
     ],
-    "documentation": "https://help.passpack.com/knowledgebase/idx.php/44/0/",
+    "custom-hardware": [
+      "Yubico OTP"
+    ],
+    "documentation": "https://passpack.com/support/getting-started/two-factor-authentication/",
+    "recovery": "https://passpack.com/support/faq/",
```

Comment:
Screenshots in the doc shows that Google Auth __or__ Yubikeys can be enabled for 2FA. Thus no U2F.